stac_mjx/controller.py

```diff
@@ -25,11 +25,38 @@
 from tqdm import tqdm
 
 # Root position (3) + quaternion (7) in qpos
-_ROOT_QPOS_LB = -jp.inf * jp.ones(7)
-_ROOT_QPOS_UB = jp.inf * jp.ones(7)
+_ROOT_QPOS_LB = -jp.inf
+_ROOT_QPOS_UB = jp.inf
+
+# mujoco jnt_type values: https://mujoco.readthedocs.io/en/latest/APIreference/APItypes.html#mjtjoint
+_JOINT_TYPE_DIMS = {
```

Comment:
Since this refers to mujoco joint types, do you mind changing this to _MJ_JOINT_TYPE_DIMS?